### PR TITLE
README update for SEDS integration

### PR DESCRIPTION
**Pull request by BearHanded**
### Description

Readme update around SEDS data ingestion

### Related ticket(s)
MDCT-2804, but no functional changes

---
### How to test

N/A

### Important updates

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [x] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,29 @@ It should be noted that while logged in as a state user, the download template b
...
+
+In the UI, when a user is filling out the form, those numbers are loaded into the table and require a user saving the submission to update the CARTS section.
+
+On the SEDS side, this topic is updated on every submission of seds data, but CARTS filters based the following:
+
+- current and prior year
+- 4th quarter data.
+- The rollover for a "new year" is October, and future submissions are not recognized until that threshold
+
+Updates outside of that time frame will need to be manually corrected in CARTS, or the integration will need to be modifed to collect data for old forms. CARTS additionally looks for the `enrollmentCounts` property which is only included in forms 21E and 64.21E (question 7), either by manual trigger or update. See SEDS files:
```
Comment (gmrabian) [inline README.md:111]: Are the `generateEnrollmentTotals` and `updateStateForms` the "manual" corrections you speak of in the first sentence, or is there other info we may need about that?

Also I'm sure you've already been through this but is there a way to design it such that we don't need manual updates?

**Review by gmrabian — DISMISSED** [review]
words

**Review by britt-mo — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (BearHanded) [reply README.md:111]: I can be more explicit here. Entries outside of that timeframe would need to be manually loaded into the enrollments table.

I think changes to this functionality would need guidance from product. Only consuming the 2 years at a time was an existing behavior within CARTS. There's definitely other ways to build it if it was a desired change.

**Comment by BearHanded** [comment]
Holding this PR to add the reference for the SEDS AL user after that merges

**Review by braxex — DISMISSED** [review]

**Review by braxex — APPROVED** [review]

**Pull request merged**